packages/lib/constants.ts

```diff
@@ -165,3 +165,11 @@ export const TRANSCRIPTION_STARTED_ICON = IS_PRODUCTION
 export const TRANSCRIPTION_STOPPED_ICON = IS_PRODUCTION
   ? `${WEBAPP_URL}/sparkles.svg`
   : `https://app.cal.com/sparkles.svg`;
+
+export const RECORDING_DEFAULT_ICON = IS_PRODUCTION
```

Comment:
Daily requires icon to be in https format so this won't be visible right now locally.